appstore/validator.go

```diff
@@ -33,47 +34,53 @@ type Client struct {
 	httpCli       *http.Client
 }
 
+var (
+	ErrAppStoreServer = errors.New("AppStore server error")
+
+	ErrInvalidJSON            = errors.New("The App Store could not read the JSON object you provided.")
+	ErrInvalidReceiptData     = errors.New("The data in the receipt-data property was malformed or missing.")
+	ErrReceiptUnauthenticated = errors.New("The receipt could not be authenticated.")
+	ErrInvalidSharedSecret    = errors.New("The shared secret you provided does not match the shared secret on file for your account.")
+	ErrServerUnavailable      = errors.New("The receipt server is not currently available.")
```

Comment:
I think that `receipt` is unnecessary.
The receipt server is not currently available.
↓
The server is not currently available.

Reply:
Thanks @kitakitabauer 
These error messages are quoted from the following official document :-)

![image](https://user-images.githubusercontent.com/4252009/74911165-67b67c80-53ff-11ea-9d28-d4674116b452.png)

https://developer.apple.com/documentation/appstorereceipts/status

Reply:
I see! It's all right.